rapid-cpio/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "dracut-cpio"
+description = "cpio archive generator for Dracut"
+authors = ["David Disseldorp"]
+license = "GPL-2.0"
+version = "0.1.0"
+edition = "2018"
+
+[dependencies]
+crosvm = { path = "third_party/crosvm" }
```

Comment:
An alternate approach is to use `crosvm = "0.1"`, then run "cargo vendor" and add the vendor subdir. Additionally you add .cargo/config with the content that cargo vendor displays and it will automatically find the location during builds. This way you can cargo vendor in the future, and it will update the crosvm sources without needing more manual interactions.

Reply:
Thanks, I wasn't aware of cargo vendor. I like that it's currently easy to see from Cargo.toml that the dependency is carried in the tree, but I suppose it makes sense to use proper vendoring. I'll make this change alongside the `lint-shell` fixes.

Reply:
I had a play with cargo vendor, but wasn't having much luck using it with `cargo --offline`, so kept things as is for now.